src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -74,11 +75,19 @@ protected Robot() {
     }
 
     private static final byte[] COMPETITION_BOT_MAC_ADDRESS = new byte[] {
-            0x00, (byte) 0x80, 0x2f, 0x33, (byte) 0x9d, (byte) 0xe7
-    };
+            0x00,
+            (byte) 0x80,
+            0x2f,
+            0x33,
+            (byte) 0x9d,
+            (byte) 0xe7 };
     private static final byte[] PRACTICE_BOT_MAC_ADDRESS = new byte[] {
-            0x00, (byte) 0x80, 0x2f, 0x28, 0x40, (byte) 0x82
-    };
+            0x00,
+            (byte) 0x80,
+            0x2f,
+            0x28,
+            0x40,
+            (byte) 0x82 };
```

Comment:
This is questionable, the one thing that'd be left to do is fix this (unless other people think it's fine)

Reply:
If there's a simple way to fix this without reformatting other parts I'd suggest fixing it, I can imagine this being a bit troublesome down the line.

Reply:
There is a way to fix it but that would disable formatting for all other array initializers. There's like zero documentation on eclipse's xml file so all I can really do is tweak the value until it works

Reply:
I'd say leave it because we want to keep areas like DrivebaseSubsystem readable